src/test/java/com/vinyl/DeleteProductFromCartTest.java

```diff
@@ -0,0 +1,133 @@
+package com.vinyl;
+
+import com.vinyl.helper.TokenHeaderHelper;
+import com.vinyl.model.Cart;
+import com.vinyl.model.Product;
+import com.vinyl.model.ProductCart;
+import com.vinyl.model.User;
+import com.vinyl.repository.CartRepository;
+import com.vinyl.repository.ProductCartRepository;
+import com.vinyl.repository.TokenRepository;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.util.Lists;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.*;
+
+public class DeleteProductFromCartTest extends LoggedInBaseIntegration {
+
+    @Autowired
+    private CartRepository cartRepository;
+
+    @Autowired
+    private TokenHeaderHelper tokenHeaderHelper;
```

Comment:
you now have the tokenHeaderHelper and the tokenRepository in the LoggedInBaseIntegration class, no need to re-declare here

Reply:
same for the  GetCartDetailsTest class